src/lib.rs

```diff
@@ -188,6 +188,8 @@ pub struct InitOptions {
 pub fn init(options: &InitOptions) -> Result<ZWaveManager,()> {
     let mut ozw_options = try!(options::Options::create("./config/", "", "--SaveConfiguration true --DumpTriggerLevel 0 --ConsoleOutput false"));
 
+    try!(options::Options::add_option_int(&mut options, "PollInterval", 60000)); //enable polling each 60 seconds
+    try!(options::Options::add_option_bool(&mut options, "IntervalBetweenPolls", true));
```

Comment:
I'm really not sure we should do this... at least not now.

This means that we wait 60sec between each poll... which means between each device too ! Whereas the default is 30000 where it tries to poll every device during this interval.  see https://github.com/OpenZWave/open-zwave/blob/master/cpp/src/Driver.cpp#L4239-L4249